docs/graph_analytics_workloads.md

```diff
@@ -1,27 +1,43 @@
 # Graph Analytics Workloads
-Broadly speaking, any kind of computation over graph data can be regarded as graph analytics. We find that computation patterns of different graph analytics vary a lot: some only involve a small number of vertices/edges, while others access a large fraction of (even all) vertices/edges of a graph. In GraphScope, we call the former as *graph traversal*, whil use the term *graph analytics* to refer to the latter, unless otherwise specified. 
+## What is Graph Analytics
+Broadly speaking, any kind of computation over graph data can be regarded as graph analytics. We find that computation patterns of different graph analytics vary a lot: some only involve a small number of vertices/edges, while others access a large fraction of (even all) vertices/edges of a graph. In GraphScope, we call the former as *graph traversal*, while use the term *graph analytics* to refer to the latter, unless otherwise specified. 
 
-Currently, there exist diverse types of graph analytics algorithms, which usually iteratively access a large fraction of (even all) vertices/edges of a graph to explore underlying insights hidden in graph data. Typical graph analytics algorithms include general analytics algorithms (e.g., PageRank, shortest path, and maximum flow), community detection algorithms (e.g., maximum clique/bi-clique, connected components, Louvain and label propagation), graph mining algorithms (e.g., frequent structure mining and graph pattern discovery). 
+Currently, there exist diverse types of graph analytics algorithms, which usually iteratively access a large fraction of (even all) vertices/edges of a graph to explore underlying insights hidden in graph data. Typical graph analytics algorithms include general analytics algorithms (e.g., PageRank, the shortest path, and maximum flow), community detection algorithms (e.g., maximum clique/bi-clique, connected components, Louvain and label propagation), graph mining algorithms (e.g., frequent structure mining and graph pattern discovery). These graph analytics algorithms have been widely applied in real scenarios. For example,  as shown in the following figure, Google uses PageRank to rank web pages in their search engine results, and the shortest path algorithm can help path planning in logistics and delivery services.

```

Comment:
We can give more details about the analytical algorithms as context and background. 
e.g., what is, what for, what's the application scenarios for 
- pagerank;
- SSSP, (shortest path)
- Louvain, (community detection)..